lightly/api/api_workflow_datasets.py

```diff
@@ -111,8 +111,8 @@ def get_datasets_by_name(
                 one dataset will be returned.
                 If True, returns only datasets which have been shared with the user. Can
                 return multiple datasets.
-                If None, returns datasets the users has access to. Can return multiple
-                datasets. Defaults to False.
+                If None, returns datasets the users has access to, including team
+                datasets. Can return multiple datasets. Defaults to False.
```

Comment:
What is the difference between `shared=True` and `shared=None`? Team datasets are shared with the user, no? This is very counterintuitive.

Reply:
Imho in the ideal case the api client would have almost no logic and just mimic the api spec parameters. But I understand that this was changed in another PR.

Reply:
Updated relevant docstrings.